sdk/core/core-websockets/src/websockets.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Types that are compatible with NodeJS Buffer.
+ */
+export type BufferLike =
+  | string
+  | Buffer
+  | DataView
+  | number
+  | ArrayBufferView
+  | Uint8Array
+  | ArrayBuffer
+  | SharedArrayBuffer
+  | readonly any[]
```

Comment:
should `any[]` be included? Feels strange if arbitrary shapes of objects are allowed. For `number` do we only allow a single byte, or treat it as multiple bytes? Also for `number[]` what about numbers that are bigger than uint8?

Reply:
This type originates in `ws` and it is currently unused. I'll revisit it once I add the actual websocket client.